src/mon/MonClient.cc

```diff
@@ -111,14 +107,17 @@ int MonClient::get_monmap_privately()
 
   ldout(cct, 10) << "have " << monmap.epoch << " fsid " << monmap.fsid << dendl;
 
+  std::random_device rd;
```

Comment:
@jdurgin

> What's the downside to using a known good method?

i think it would be better if we can maintain less code by leveraging existing and well-documented / tested library.

> it seems easy to avoid potential bugs in ports to other platforms 
 on platforms w/o `rdrand`, libstdc++ reads from `/dev/urandom` instead so we already avoid potential bugs by using `std::random_device`, right?

> or using a different stdlib

libc++ use `arc4random()` or `nacl_secure_random()` or `/dev/urandom` or `rand_s()` on windows with different settings. all of them are non-deterministic.

i do care about portability just like you do, but i also want to avoid re-inventing the wheel to archive this. even if the wheel is already in our arsenal.